src/libraries/react-calendar/components/CalendarToolbar.tsx

```diff
@@ -1,260 +1,153 @@
+})
 
 const languageOptions = [
     {
         value: 'en-US',
         label: 'English',
     },
-    {
-        value: 'pt-BR',
-        label: 'Português',
-    },
 ]
 
-function CalendarToolbar(props: any) {
-    const classes = useStyles()
-    const {
-        // open,
-        // handleDrawerOpen,
-        // handleDrawerClose,
-        changeLanguage,
-        goToToday,
-        next,
-        previous,
-        intl,
-        // match,
-    } = props
-
-    const { stateCalendar, setStateCalendar } = useContext(CalendarContext)
-    const { selectedDate, locale, i18nLocale, layout } = stateCalendar
+interface CalendarToolbarProps {
+    goToToday: () => void
+    next: () => void
+    previous: () => void
+}
 
-    return useMemo(() => {
-        const setLayout = (props: any) => {
-            const { option } = props
-            setStateCalendar({ ...stateCalendar, layout: option })
-        }
+function CalendarToolbar({
+    goToToday,
+    next,
+    previous,
+}: CalendarToolbarProps) {
 
-        const weeks = getWeekDays(selectedDate, 7)
-        const selectedWeekIndex = getSelectedWeekIndex(selectedDate, weeks, 0)
-        const selectedWeek = weeks[selectedWeekIndex]
+    const { stateCalendar, setStateCalendar } = useContext(CalendarContext)
+    const { selectedDate, layout } = stateCalendar
 
-        const firstDayOfWeekMonth = format(selectedWeek[0], 'MMM', { locale: locale })
-        const lastDayOfWeekMonth = format(selectedWeek[6], 'MMM', { locale: locale })
-        const firstDayOfWeekYear = format(selectedWeek[0], 'YYYY', { locale: locale })
-        const lastDayOfWeekYear = format(selectedWeek[6], 'YYYY', { locale: locale })
+    const setLayout = (layout: Layout) => {
+        setStateCalendar({ ...stateCalendar, layout })
+    }
 
-        const showMonthsAndYears =
-            layout === 'week' &&
-                (firstDayOfWeekMonth !== lastDayOfWeekMonth && firstDayOfWeekYear !== lastDayOfWeekYear)
-                ? `${firstDayOfWeekMonth} ${firstDayOfWeekYear} - ${lastDayOfWeekMonth} ${lastDayOfWeekYear}`
-                : false
-        const showMonthsAndYear =
-            !showMonthsAndYears && layout === 'week' && firstDayOfWeekMonth !== lastDayOfWeekMonth
-                ? `${firstDayOfWeekMonth} - ${lastDayOfWeekMonth} ${firstDayOfWeekYear}`
-                : false
-        const showMonthAndYear = !showMonthsAndYear
-            ? format(selectedDate, 'MMMM YYYY', { locale: locale })
-            : false
+    const showMonthsAndYears = format(selectedDate, 'MMMM YYY')
 
-        return (
-            <div
-                // position='fixed'
-                className={clsx(classes.root, classes.appBar)}
-            // className={clsx(classes.root, classes.appBar, {
-            //     [classes.appBarShift]: open,
-            // })}
-            // color='default'
-            >
-                <Toolbar>
-                    {/* <IconButton
+    return (
+        <RootWrapper>
```

Comment:
You didn't include `classes.appBar` in `RootWrapper` component.

Reply:
appBar class has describes transition for the action I deleted

Reply:
Oken, noted.